### ISSUE TITLE
Insert table works, edit table does not work

### ISSUE THREAD

**Issue by fpoto** [cosmocode/edittable#110]
Using Dokuwiki 2016-06-26a, I install the plugin and I add a new table using the button. The table editor works as expected.

I save the page and an Edit button appears under the table. When I press the button, I get an edit window with just the Save, Preview, Cancel buttons and on top of them a link to Media Files. The editor does not pop up.

For some reason I cannot upload a screen shot, but you can see it at http://pot.potorti.it/tmp/dokuwiki-bug-No.110.png

**Comment by hick** [comment]
yes I got the same result.  both latest edittable and dokuwiki.org

**Comment by splitbrain** [comment]
please check your JavaScript error console and report any errors seen there.

**Comment by chrishem** [comment]
I'm having the same problem.
Here's what was in my Javascript Console:
Uncaught ReferenceError: Invalid left-hand side expression in postfix operation js.php:23

**Comment by mirgcire** [comment]
I'm having the same problem and not seeing any javascript errors in the console.
Also I noticed that the table button is not appearing in the edit window menu bar.

**Comment by Sarabjeet76** [comment]
I am having same problem and not able to use edit table plugin

What is the solution to this problem

Here's what was in my Javascript Console:
Uncaught ReferenceError: Invalid left-hand side expression in postfix operation js.php:23

**Comment by micgro42** [comment]
What DokuWiki version and which Browser are you using?

**Comment by bby-bishopclark** [comment]
> What DokuWiki version and which Browser are you using?
```
dokuwiki-20180422b-3.el7.noarch
dokuwiki-plugin-c3chart-0.2.0-0.1.noarch
dokuwiki-plugin-csv-20180427-0.1.f12410b.noarch
dokuwiki-plugin-cellbg-1.0.1-0.1.b29e555.noarch
dokuwiki-plugin-edittable-20200511-0.1.1917599.noarch
dokuwiki-plugin-imgpaste-20180503-0.1.noarch
```
```
Google Chrome is up to date
Version 86.0.4240.111 (Official Build) (64-bit)
```
> Please note that edittable is incompatible with Hrun.

I need to find my decoder ring.

**Comment by bby-bishopclark** [comment]
Hey Folks, 

I just bulldozed and redid my tst DW box.  I'm running dokuwiki-plugin-edittable-20200511-0.1.1917599.noarch again, and it seems like it wants to work.  Maybe re-test on 2020-05-11 or later?
